### PR TITLE
Added MySQL database configuration to the docs

### LINKED ISSUE #1332: Allow Metabase to use MySQL as the application database
Right now [the documentation](https://github.com/metabase/metabase/blob/master/docs/operations-guide/running-the-metabase-jar-file.md#the-metabase-application-database) reads:

    The default settings use an H2 database, but other options are available.

H2 and Postgresql examples are given, but what about other RDMS vendors? The documentation should one of:

* Metabase only supports H2 and Postgresql, and nothing else

* Metabase supports H2 and Postgresql, but we haven't tested other providers.

* Metabase should support most common RDMS vendors. H2 and Postgresql have been tested.

### PR DESCRIPTION
**Pull request by lukaswelte**
Documents changes made in #1370 that resolves #1332

### CODE REVIEW

File: docs/operations-guide/running-the-metabase-jar-file.md [hunk 1]
```diff
@@ -86,6 +86,19 @@ You can change the application database to use Postgres using a few simple envir
 
 This will tell Metabase to look for its application database using the supplied Postgres connection information.
 
+#### [MySQL](http://www.mysql.com/)
+You can change the application database to use MySQL using a few simple environment variables. For example:
+
+    export MB_DB_TYPE=mysql
+    export MB_DB_DBNAME=metabase
+    export MB_DB_PORT=5432
```
Comment (agilliland) [inline docs/operations-guide/running-the-metabase-jar-file.md]: the default port for MySQL is 3306.  lets fix that and then this looks g2g :+1:

Comment (lukaswelte) [inline docs/operations-guide/running-the-metabase-jar-file.md]: @agilliland I changed it to be the default MySQL port. Thanks for the mention!

**Comment by TharunPonnaganti** [comment]
is there any video on how to connect mysql database to metabase

**Comment by flamber** [comment]
@TharunPonnaganti Please use the forum for questions and troubleshooting: https://discourse.metabase.com/

**Pull request merged**